util/p9/backup

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+
+timestamp=$(date +"%Y-%m-%d_%H-%M-%S")
+tar czf ./gplt.${timestamp}.tar.gz ./gatling-puppet-load-test
```

Comment:
What directory is this command relative to? Should there be an environment variable set to provide the expected root path for this command?

Reply:
It assumes that you are in an instance directory (i.e. '321a') that contains a gatling-puppet-load-test directory. When these are converted into bolt tasks they will be relative to the gatling-puppet-load-test directory that contains them.

Reply:
Added [docs/slv_environment_setup.md](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md) with a [Utility scripts](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md#utility-scripts) section.